src/Shopsys/ShopBundle/Form/Admin/CategoryFormTypeExtension.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: miroslavstopka
+ * Date: 7/30/18
+ * Time: 8:28 AM
+ */
+
+namespace Shopsys\ShopBundle\Form\Admin;
+
+use Ivory\CKEditorBundle\Form\Type\CKEditorType;
+use Shopsys\FormTypesBundle\MultidomainType;
+use Shopsys\FrameworkBundle\Form\Admin\Category\CategoryFormType;
+use Shopsys\FrameworkBundle\Form\FormRenderingConfigurationExtension;
+use Shopsys\FrameworkBundle\Form\GroupType;
+
```

Comment:
hmm, wonder whether this blank line between use statements passes the standards

Reply:
You are right, I have removed this empty line